cypress/pages/shoppingPage.ts

```diff
@@ -0,0 +1,25 @@
+const products = Cypress.env("Products");
+
+class CategoriesPage {
+  constructor(){
+    this.elementSelector = ".card-title",
+    this.containerElement = "#tbodyid",
+    this.addToCart = ".btn-lg",
+    this.addToCartBtn = "Add to cart"
+    this.navBarSelector = ".nav-link",
+    this.home = "Home",
+    this.categoriesSelector = ".list-group-item"
+  }
+
+  addElementsToCart(elements) {
```

Comment:
Ale, muy muy bien hecha esta función, sin embargo recuerda el principio de responsabiliad única... en esta sola función estas: navegando al elemento, verificando el elemento y añadiendo el elemento... todo en una sola función... sería más práctico que en el test se vea el paso a paso de lo que estás haciendo, y crees solo la función de añadir al carrito o de verificar el título de este...

Reply:
Se divide la función en: 

- Add products to cart
- Verify that Product Tittle corresponds